### ISSUE TITLE
Change default text in "About this collection"

### ISSUE THREAD

**Issue by VictorHarbo** [netarchivesuite/solrwayback#354]
The `About this collection`-button contains text that is specific for the Danish Netarkiv. As SolrWayback is used by many different institutions and persons, this file should contain boilerplate text in SolrWayback instead of the actual Netarkivet text as pointed out by a participant at today's SolrWayback workshop at IIPC.

**Comment by thomasegense** [comment]
It can be customized. It is in the propertyfile

**Comment by VictorHarbo** [comment]
> It can be customized. It is in the propertyfile

Yes. The point is to change the default to a generic text so that people creating SolrWayback installations around the globe are not accidentally referring to information on the Danish Netarkiv.

**Comment by thomasegense** [comment]
I understand. Should be quick fix

**Comment by joncto** [comment]
Even with a new, generic text, maybe one could add a sentence in the documentation that users should edit the content of `about_collection.txt`, found in `../solrwayback_package_4.4.1/apache-tomcat-8.5.60/webapps/solrwayback/WEB-INF/classes`?
